stable/keycloak/templates/configmap.yaml

```diff
@@ -20,8 +20,8 @@ data:
     echo 'Running custom pre-start script...'
 {{ . | indent 4 }}
   {{- end }}
-
-    exec /opt/jboss/tools/docker-entrypoint.sh -b 0.0.0.0 {{ .Values.keycloak.extraArgs }}{{- if $highAvailability }} -c standalone-ha.xml{{ else }} -c standalone.xml{{ end }}
+    # Moving standalone.xml earlier as it get moved to next line when templating this chart
+    exec /opt/jboss/tools/docker-entrypoint.sh -b 0.0.0.0 {{ if $highAvailability }} -c standalone-ha.xml{{ else }} -c standalone.xml{{ end }} {{ .Values.keycloak.extraArgs }}    
```

Comment:
I fail to see what the issue is and cannot reproduce it. Don't you just move the extra args to the end? Why would that change anything?

Reply:
@unguiculus The issue is if I add multiple extraArgs like in values.yaml if I add 
```
  extraArgs: >
    -Djgroups.bind_addr=global
    -Dkeycloak.migration.strategy=IGNORE_EXISTING    
    -Dkeycloak.migration.action=import
```
and I templatize the keycloak chart, it adds the standalone.xml statement in next line like 
```
1. exec /opt/jboss/tools/docker-entrypoint.sh -b 0.0.0.0 -Djgroups.bind_addr=global -Dkeycloak.migration.strategy=IGNORE_EXISTING     -Dkeycloak.migration.action=import
2. -c standalone.xml
```

Reply:
You need to use this:

```yaml
  extraArgs: >-
    -Djgroups.bind_addr=global
    -Dkeycloak.migration.strategy=IGNORE_EXISTING    
    -Dkeycloak.migration.action=import
```

Note the hyphen after `>`.